.github/workflows/staging-deploy.yml

```diff
@@ -0,0 +1,66 @@
+name: staging deployment
+
+on: pull_request
+
+jobs:
+  test-and-lint:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.9.16
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.9.16'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Lint with flake8
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 pages app.py utils index.py predeploy_update_data.py tasks.py --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
Can remove these specific files and just run `flake8` against all python files in the repo

Reply:
oops yup